src/trials.jl

```diff
@@ -465,13 +466,7 @@ function Base.show(io::IO, ::MIME"text/plain", t::Trial)
         end
     end
 
-    function remtrailingzeros(timestr)
-        if ! isnothing(match(r"\.0+$", timestr))
-            replace(timestr, r"\.0+$" => "")
-        else
-            replace(timestr, r"(\.\d+?)0+$" => s"\1")
-        end
-    end
+    remtrailingzeros(timestr) = replace(timestr, r"\.?0+ " => " ")
```

Comment:
What's with the space in this regexp?

Reply:
Oh, you might have looked at this before I posted my comment that explains why I changed this function. To recap: `timestr` includes units (e.g., `1.010 ns`), so we need to anchor to a space instead of the end of line. In any case, the spaces on each side act as a positive lookahead, so this is equivalent to `r"\.?0+(*pla: )" => ""`. Without them the function would revert to unconditionally removing the decimal point and right-adjacent zeros (which might not be trailing).

Reply:
Ah right, I just thought this was a random test.

Reply:
I did look at this before reading your comment, this seems sensible.